frontend/src/components/ImplementationReportView/BadgeFormats.tsx

```diff
@@ -0,0 +1,65 @@
+export interface BadgeFormatOption {
+  type: string;
+  renderDisplay: (badgeURI: string) => JSX.Element;
+  generateCopyText: (badgeURI: string) => string;
```

Comment:
What made you split this into its own property? It seems like it only adds the possibility for subtle bugs -- specifically us showing something to the user that's different than what we copy?

Reply:
I thought about splitting `renderDisplay` and `generateCopyText` property specially and only to show `rSt` format. Other formats were easy to manage with a string property. Also made `BadgeFormatOptions` array homogeneous and developer friendly.

Reply:
What's special about rst?

Reply:
To format `rSt` on render I used `<pre>` tag.  So any user can copy formatted badge with cursor also. I took the inspiration from https://shields.io/badges component.
![image](https://github.com/bowtie-json-schema/bowtie/assets/43758514/38009352-71e3-43e9-8a0a-6228e9db7dbb)

Reply:
I don't follow what's different about reST than any other format there.

Reply:
Yes but of course everything we show should be rendered the way we intend it to be rendered, not just reST.